packages/otelbin/src/app/s/[id]/metadataUtils.test.ts

```diff
@@ -45,80 +44,6 @@ describe("extractComponents", () => {
 	});
 });
 
-describe("calcScale", () => {
```

Comment:
**Question:** This test got removed, but the function continues to exist. Why remove it?

Reply:
The entire logic for calculating the scale has been revised. New tests will be added.